### DISCUSSION TITLE
Typos on Wiki of Contributing guidelines

### DISCUSSION THREAD

**Discussion by vandan-revanur** [rdkit/rdkit#6243] [General]
## Typo 1:

ansewr -> answer

## Typo 2:

Under the heading : Are you a Python or C++ developer?

A currently non existent tag: [small change](https://github.com/rdkit/rdkit/issues?q=is%3Aopen+is%3Aissue+label%3A%22small+change%22) is mentioned.

When new contributors click on this they are under the wrong impression that there is nothing for them to contribute since there are no issues under that non-existing tag.

It should instead be: [good for beginners](https://github.com/rdkit/rdkit/labels/good%20for%20beginners)

**Comment by vandan-revanur** [comment]
@greglandrum could you please have a look and correct these typos?

**Reply by greglandrum** [reply]
Done. Thanks for pointing that out
